tests/tests_unit.yml

```diff
@@ -11,6 +11,17 @@
         src: test_network_connections.py
         dest: /tmp/test-unit-1/
 
+    - file:
+        state: directory
+        dest: /tmp/test-unit-1/helpers
+
+    - copy:
+        src: "{{ item }}"
+        dest: /tmp/test-unit-1/helpers
+      with_fileglob:
+        - helpers/*
+
+    - shell: chmod -R a+x /tmp/test-unit-1/helpers
 
```

Comment:
Is that really necessary? The file is executable (should be, I suppose, if you want to run the test locally) and I believe `copy` preserves the permissions (if not, use the `mode` parameter).

Reply:
Good catch, I will fix it.